src/renderer/ui/components/settings/ToggleableOption.js

```diff
@@ -17,7 +18,8 @@ class SettingsCheckbox extends Component {
   render() {
     return (
       <Checkbox
-        label={this.props.label}
+        label={[this.props.label, this.props.hintLabel
+          ? <span id="checkbox-hint" style={{ display: 'block', fontSize: '0.75em', color: 'gray' }}>{this.props.hintLabel}</span> : null]}
```

Comment:
Why does this span have an ID?  ID's are by definition supposed to be unique when we could have multiple of these on the page.

It would make more sense for this to be a class and then to apply the styling through the LESS stylesheets